README.md

```diff
@@ -19,8 +19,7 @@ Include configuration of code and style used in MagicMirror core. Also to test t
 
 ## Installation
 
-`curl -sL https://raw.githubusercontent.com/roramirez/MagicMirror-Module-Template/master/create_module.sh | bash`
-
+`bash -c "$(curl -sL https://raw.githubusercontent.com/roramirez/MagicMirror-Module-Template/master/create_module.sh)"`
```

Comment:
Old version could not be executed (since input stream piped to bash is interpreted as user input).